west.yml

```diff
@@ -41,6 +43,10 @@ manifest:
     - name: nrfxlib
       path: nrfxlib
       revision: d98b59bdff2b533f5afb7bdad229eaeb4a931882
+    - name: cmock
+      path: cmock
```

Comment:
This should probably be somewhere deeper like `test/cmock`. Otherwise we will end up with a realy wide root folder. @SebastianBoe @tejlmand thoughts?

Reply:
No comment.

Reply:
Agree with @carlescufi .

Another reason why I would like this further down, is to make sure it the the most important part of NCS is located at toplevel, and then supporting tools, such as test tools are further down.
There may be more such tools in future.

Upstream Zephyr start to have `modules/` should we place it there instead of `test` ?